fetcher/responses/nodestats_response.go

```diff
@@ -165,6 +169,12 @@ type SinglePipelineResponse struct {
 				WithErrors int            `json:"with_errors"`
 				Responses  map[string]int `json:"responses"`
 			} `json:"bulk_requests"`
+			Flow struct {
```

Comment:
This struct also exists in the "Filters" (but not codecs or inputs) object in the data returned from _node/stats

Not sure if important

Reply:
all right, added it to plugins, but not to metrics (at least for now, not sure whether somebody needs this stats, but should be quite easy to add it in the future)